src/ontology/edits/oeo-physical.omn

```diff
@@ -9963,6 +9966,226 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
         OEO_00140104
     
     
+Class: OEO_00310032
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A thermo-chemical heat storage (TCS) is a thermal energy storage object that stores thermal energy through reversible exotherm/endotherm chemical reaction with thermo-chemical materials (TCM)."@en,
```

Comment:
I'd suggest to add TCS as alternative term.

Reply:
Are TCM part of the OEO?

Reply:
Ok many thanks for the fixes! Many classes still have them, so I assumed that this would be the normal style. 

Currently, TCM are not part of the OEO.